data/restli-csharp/datagenerator/src/test/GeneratedDataTemplate/com/linkedin/restli/datagenerator/integration/ComplexRequiredRecord.cs

```diff
@@ -0,0 +1,198 @@
+using System.Collections.Generic;
+using System;
+
+// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
+// Generated from com/linkedin/restli/datagenerator/integration/ComplexRequiredRecord.pdsc
+
+namespace com.linkedin.restli.datagenerator.integration
```

Comment:
When I reviewed to the bottom of this RB, I noticed that we have checked in such generated files twice. Since they are already in `RestliDataIntegration`, we should not check in this `generatedDataTemplate` directory at all, put this directory in `.gitignore`.